core/jobs_test.py

```diff
@@ -789,9 +789,26 @@ def test_each_one_off_class_is_subclass_of_base_job_manager(self):
         for klass in jobs_registry.ONE_OFF_JOB_MANAGERS:
             self.assertTrue(issubclass(klass, jobs.BaseJobManager))
 
+    def test_is_abstract_method(self):
+        class TestMockAbstractClass(jobs.BaseJobManager):
+            """A sample Abstract Class."""
+            pass
+
+        mock_abstract_base_class = [
+            TestMockAbstractClass
+        ]
+        abstract_list_swap = self.swap(
```

Comment:
`abstract_list_swap` -> `abstract_class_list_swap`

Reply:
Done. Instead of renaming, I removed this variable as suggested in a earlier comment.